src/gefs_6h_ave_1mem.fd/gefs_6h_ave_1mem.f90

```diff
@@ -0,0 +1,538 @@
+      program gefs_6h_ave_1mem
```

Comment:
This is a f90 code, so it does not need to have 6 spaces. The modern compiler will understand.